python/hail/expr/functions.py

```diff
@@ -1869,7 +1869,11 @@ def is_transition(ref, alt) -> BooleanExpression:
     -------
     :class:`.BooleanExpression`
     """
-    return _func("is_transition", tbool, ref, alt)
+    return is_snp(ref, alt) & (
```

Comment:
The scala function accounts for snps that aren't just a single character long--e.g. `[AAAT, AAGT]`.